docs/posts/annotations.md

```diff
@@ -36,10 +35,7 @@ There are two options for updating draft annotations.
 
 ### Autodetect annotations csv
 
-Cellxgene can automatically find and reload your draft annotations in editable mode like so:  
-```
-annotations-directory$ cellxgene launch mydata.h5ad --experimental-annotations
-```
+Cellxgene will automatically find and reload your draft annotations in editable mode.
```

Comment:
Nit:  what is `editable mode`?   Maybe "If annotations are enabled, cellxgene will automatically find and reload your draft annotations"?

Also, this is only true if the user has not specified an alternative annotations file or directory.

Reply:
ps. I'm fine if this is refined post-PR